### PR TITLE
OPS-456 move cookie secret to environment var

### PR DESCRIPTION
**Pull request by pkleybolte**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/OPS-456

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: config/default.schema.json [hunk 1]
```diff
@@ -274,6 +274,10 @@
 				"EXPIRES_SECONDS"
 			]
 		},
+		"COOKIE_SECRET": {
+			"type": "string",
+			"description": "Secret used to sign the session ID cookie"
+		},
```
Comment (Parnswir) [inline config/default.schema.json:280]: to not break anything (and according to Jan's comment in the ticket), this should default to 'secret'

File: config/default.json [hunk 2]
```diff
@@ -13,5 +13,6 @@
 		"SECURE": false,
 		"EXPIRES_SECONDS": 2592000000
 	},
+	"COOKIE_SECRET": "ihasDF(=4jA4aF",
```
Comment (Parnswir) [inline config/default.json:16]: Is this necessary when using the Configuration?

File: config/default.schema.json (hunk 1 above)
Comment (pkleybolte) [reply config/default.schema.json:280]: I talked to Jan. He agreed that it is safer to use a new password. Maybe some old sessions might invalidate, but that's not a problem.

File: config/default.json (hunk 2 above)
Comment (pkleybolte) [reply config/default.json:16]: Could be moved to default.schema.json. We agreed to leave it here to "force" to set it explicitly in the instances and not use the default, that is publicly readable by everybody cloning the repo.

**Pull request merged**